lib/srte.h

```diff
@@ -19,6 +19,25 @@ enum zebra_sr_policy_status {
 	ZEBRA_SR_POLICY_DOWN,
 };
 
+/* SR types. */
+enum sr_types {
+	ZEBRA_SR_LSP_NONE = 0,	/* No LSP. */
+	ZEBRA_SR_LSP_SRTE = 1,	/* SR-TE LSP */
+	ZEBRA_SR_SRV6_SRTE = 2, /* SRv6 SID List*/
+};
+
+static inline enum lsp_types_t lsp_type_from_sr_type(enum sr_types sr_type)
+{
+	switch (sr_type) {
+	case ZEBRA_SR_LSP_SRTE:
+		return ZEBRA_LSP_SRTE;
+	case ZEBRA_SR_LSP_NONE:
+	case ZEBRA_SR_SRV6_SRTE:
+	default:
```

Comment:
remove the default this is an enum and all the values should be accounted for in the switch statement.  Leaving a default allows future changes to the enum to be masked here and the developer will miss it.

Reply:
fixed